src/crimson/osd/pg_shard_manager.cc

```diff
@@ -3,12 +3,6 @@
 
 #include "crimson/osd/pg_shard_manager.h"
 
-namespace {
```

Comment:
I know Sam is working in this area right now, so my guess would be we'll start using `logger()` soon.

Reply:
Yeah, let's leave that for now.

Reply:
Or, actually, you can go ahead and remove it -- I'll just readd it once it's got a user.